### DISCUSSION TITLE
LMN3 on Raspberry pi4 B ?

### DISCUSSION THREAD

**Discussion by MichaelDouaud** [FundamentalFrequency/.github#73] [General]
Hi, 
i am a French Musician here and i'm looking for LMN3 solution to built my own portable sampler/daw.
Question is: can i use LMN 3 sources in my hardware configuration ? i have Raspberry pi4 B and HIFI DAC ADC pro and 7 inch touch screen.

Thank's for reading, LMN3 is fantastic job

Michaël

**Comment by stonepreston** [comment]
you would have to build a custom case to fit that screen/hat configuration. Never used that hat before, but it should work assuming the DAC shows up as an output like any other audio output would (the LMN-3 can use the DAC output, but it currently does not support inputs)
